src/main.rs

```diff
@@ -68,6 +74,7 @@ fn main() {
 
   log::set_logger(&LOGGER).unwrap();
   let args = env::args().collect();
+  let args2: Vec<String> = env::args().collect();
```

Comment:
an alternative here is to do:
```
let start_repl = args.len() == 1
```
(You'll need to set `args: Vec<String>` above.)

Reply:
okay

Reply:
I take this back, in future you're going to have to parse args. e.g. `-D`.